omnipaxos/src/messages.rs

```diff
@@ -50,15 +50,29 @@ pub mod sequence_paxos {
         pub n: Ballot,
         /// The latest round in which an entry was accepted.
         pub n_accepted: Ballot,
+        /// The decided index of this follower.
+        pub decided_idx: usize,
+        /// The log length of this follower.
+        pub accepted_idx: usize,
+        /// The log update which the leader applies to its log in order to sync
+        /// with this follower (if the follower is more up-to-date).
+        pub log_sync: Option<LogSync<T>>,
+    }
+
+    /// An update that a replica applies to its log in order to sync with another replica's log.
+    #[derive(Clone, Debug)]
+    #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+    pub struct LogSync<T>
```

Comment:
Move to `util.rs` (next to AcceptedMetaData and similar structs) as it's not a message.